apps/webpack.js

```diff
@@ -1,7 +1,12 @@
 var _ = require('lodash');
 var webpack = require('webpack');
 var path = require('path');
-var config = {
+var LiveReloadPlugin = require('webpack-livereload-plugin');
+
+var AUTO_RELOAD = ['true', '1'].indexOf(process.env.AUTO_RELOAD) !== -1;
```

Comment:
Should this bit of logic be shared between `webpack.js` and `Gruntfile.js`? We use boolean environment variable flags all over the place, so we should be consistent with how we handle them. i.e. does just the presence of the flag (regardless of the value) imply that the flag is "on"? Or does the environment variable need to be exactly 1 or true, and not True or Yes or No?